concepticondata/conceptlists/Gabelentz-1892-1074.tsv

```diff
@@ -0,0 +1,1075 @@
+Gabelentz-1892-1074-947	947	Bleiben	to stay	VIII. Verba	E. Dasein, verweilen	987	REMAIN
+Gabelentz-1892-1074-948	948	Wohnen	to reside, to dwell	VIII. Verba	E. Dasein, verweilen	1099	RESIDE
+Gabelentz-1892-1074-949	949	Sitzen	to sit (be sitting)	VIII. Verba	E. Dasein, verweilen	1416	SIT
+Gabelentz-1892-1074-950	950	Stehen	to stand (be standing)	VIII. Verba	E. Dasein, verweilen	1442	STAND
+Gabelentz-1892-1074-951	951	stellen	to put something somewhere	VIII. Verba	E. Dasein, verweilen		
+Gabelentz-1892-1074-952	952	Biegen	to bend	VIII. Verba	E. Dasein, verweilen	2224	BEND (SOMETHING)
+Gabelentz-1892-1074-953	953	Legen	to lay something somewhere	VIII. Verba	E. Dasein, verweilen		
+Gabelentz-1892-1074-954	954	Hängen (intransitiv)	to hang (intransitive)	VIII. Verba	E. Dasein, verweilen	2884	HANG
+Gabelentz-1892-1074-955	955	hängen (transitiv)	to hang (transitive)	VIII. Verba	E. Dasein, verweilen	357	HANG UP
+Gabelentz-1892-1074-956	956	Kleben	to stick	VIII. Verba	E. Dasein, verweilen	254	STICKING
+Gabelentz-1892-1074-957	957	Geschehen	to happen	VIII. Verba	E. Dasein, verweilen	248	HAPPEN
+Gabelentz-1892-1074-958	958	Vergeben	to forgive	VIII. Verba	E. Dasein, verweilen	1528	FORGIVE
+Gabelentz-1892-1074-959	959	Fehlen, nicht dasein	to be missing	VIII. Verba	E. Dasein, verweilen		
+Gabelentz-1892-1074-960	960	Abreissen, trennen	to tear apart	VIII. Verba	F. Andere Verba (alphabetisch geordnet)	1735	TEAR (SHRED)
+Gabelentz-1892-1074-961	961	Abwischen	to wipe	VIII. Verba	F. Andere Verba (alphabetisch geordnet)	1454	WIPE
+Gabelentz-1892-1074-962	962	Anfangen	to begin	VIII. Verba	F. Andere Verba (alphabetisch geordnet)	3447	BEGIN OR START
+Gabelentz-1892-1074-963	963	Anklopfen	to knock	VIII. Verba	F. Andere Verba (alphabetisch geordnet)	2396	KNOCK
+Gabelentz-1892-1074-964	964	Anrühren	to touch	VIII. Verba	F. Andere Verba (alphabetisch geordnet)	1892	TOUCH
+Gabelentz-1892-1074-965	965	Anzünden	to set something on fire	VIII. Verba	F. Andere Verba (alphabetisch geordnet)	737	LIGHT (IGNITE)
+Gabelentz-1892-1074-966	966	Arbeiten	to work	VIII. Verba	F. Andere Verba (alphabetisch geordnet)	1086	WORK (ACTIVITY)
+Gabelentz-1892-1074-967	967	Aufrollen, rollen	to roll	VIII. Verba	F. Andere Verba (alphabetisch geordnet)	827	ROLL
+Gabelentz-1892-1074-968	968	Aufsetzen (den Hut)	to put on (a hat)	VIII. Verba	F. Andere Verba (alphabetisch geordnet)	158	PUT ON
+Gabelentz-1892-1074-969	969	Ausbreiten	to spread	VIII. Verba	F. Andere Verba (alphabetisch geordnet)	38	SPREAD OUT
+Gabelentz-1892-1074-970	970	Ausstrecken	to stretch out	VIII. Verba	F. Andere Verba (alphabetisch geordnet)	214	STRETCH
+Gabelentz-1892-1074-971	971	Bauen	to build	VIII. Verba	F. Andere Verba (alphabetisch geordnet)	1840	BUILD
+Gabelentz-1892-1074-972	972	Bedecken	to cover	VIII. Verba	F. Andere Verba (alphabetisch geordnet)	761	COVER
+Gabelentz-1892-1074-973	973	bekleiden, anziehen	to dress	VIII. Verba	F. Andere Verba (alphabetisch geordnet)	814	DRESS UP
```

Comment:
That sounds like it refers to dressing someone else.

Reply:
Because he didn't add the reflexive "sich"? But I feel like he left that out in some places in the list...

Reply:
Yes, there are a few cases which look like a "sich" is simply missing. But there are also some "sich" in the list. So I'm not sure how to interpret this.

Reply:
I think we can leave it as is. At least the English glosses mapped to the concept imply that it refers to both _dress someone_ and _dress oneself_.